src/main/scala/org/globalforestwatch/summarystats/treecoverloss/TreeLossExport.scala

```diff
@@ -13,12 +14,25 @@ object TreeLossExport extends SummaryExport {
     val spark = summaryDF.sparkSession
     import spark.implicits._
 
-    val includePrimaryForest: Boolean =
-      getAnyMapValue[Boolean](kwargs, "includePrimaryForest")
+    val maybeContextualLayers: Option[NonEmptyList[String]] =
```

Comment:
is it possible to make contextualLayers just List[String] (that defaults to empty) instead of Option[NonEmptyList[String]]? I feel like that would simplify this code a lot

Reply:
Unfortunately not.
The `Decline` module we use for parsing arguments uses a `NonEmptyList` for multiple options. I would either have to use a List with and empty String `""` as value or use `.orNone` and then the `Option` type

Reply:
Used empty string non the less to make code easier to read